src/nodes/export-declaration.ts

```diff
@@ -30,27 +32,35 @@ export default class ExportDeclaration extends Node<ExportDeclarationType> {
 
       // split exports into type and value exports
       const rawElements = this.raw.exportClause.elements;
-      let typeExports, valueExports;
-      if (isTypeImport) {
-        typeExports = rawElements;
+      let typeExports: ExportSpecifier[];
+      let valueExports: ExportSpecifier[];
```

Comment:
Thanks for splitting this out. Multiple variable declarations with the same `let/var/const` keyword might be my single least favorite js feature, which is saying something!